### PR TITLE
chore: modifying package metadata

### PR DESCRIPTION
**Pull request by gratcliff**
| 🚥 Resolves ISSUE_ID |
| :------------------- |

## 🧰 Changes

- Updating package name to include `@readme`, inline with `npm`.
- Exposing a typescript `.d.ts` file for our default exposed function.

## 🧬 QA & Testing

Provide as much information as you can on how to test what you've done.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,9 +1,10 @@
 {
-  "name": "http-headers",
+  "name": "@readme/http-headers",
   "version": "1.0.0",
   "description": "Retrieve HTTP Header Descriptions ",
```
Comment (kanadgupta) [inline package.json:4]:
```suggestion
  "description": "Retrieve HTTP Header Descriptions",
```

File: package.json [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
   "main": "dist/index.js",
   "type": "module",
```
Comment (kanadgupta) [inline package.json:6]: I'm guessing this is not supposed to be ESM, right?

```suggestion
```

File: package.json
Comment (kanadgupta) [inline package.json:1]: I'd also add an `engines` field somewhere, and set the value to `>=16` (or whatever you're running tests against)!

File: package.json [hunk 3]
```diff
@@ -1,9 +1,10 @@
 {
-  "name": "http-headers",
+  "name": "@readme/http-headers",
   "version": "1.0.0",
```
Comment (kanadgupta) [inline package.json:3]: up to you here but if you wanted to do a pre-release version just to make sure it publishes properly:

```suggestion
  "version": "0.0.1",
```

File: package.json
Comment (kanadgupta) [reply package.json:1]: Oh, and run `npm i` from the root so the changes here get surfaced in your lockfile!

**Review by kanadgupta — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (gratcliff) [reply package.json:6]: It is, unfortunately this package is `esm` only, as the sum of its parts.

File: package.json (hunk 3 above)
Comment (gratcliff) [reply package.json:3]: Yeah, good call.

**Pull request merged**